src/Client/Phiremock.php

```diff
@@ -209,4 +212,15 @@ private function getRequestBuilder()
 
         return $this->simpleRequestBuilder;
     }
+
+    private function registerAnnotationsLoader()
+    {
+        if (file_exists(__DIR__ . '/../../vendor/autoload.php')) {
```

Comment:
I guess this is not needed, only the autoloader in the annotation registry. My suggestion is to not register in this class but add documentation in the readme for the application to use this to register the autoloader in their bootstrap.

Reply:
It's possible, but in this way `\Codeception\Module\Phiremock` from `mcustiel/phiremock-codeception-extension` should register annotations autoload to have ability be extended with list expectations method, and it will be looks like that with greater tree depth only.

So I think it is better place to implement it.